docs/Concepts/Proof-of-Stake.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Ethereum proof of stake
+---
+
+# Proof of stake
+
+[The Merge](Merge.md) transitions Ethereum Mainnet to
+[proof of stake (PoS)](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/) consensus.
+
+In Ethereum's PoS, you must run a [full node](Merge.md#execution-and-consensus-clients) and
+[stake 32 ETH](https://ethereum.org/en/staking/) to become a validator.
+
+!!! important
+
+    Withdrawing staked ETH isn't yet supported and will be included in a separate upgrade following The Merge.
+
+!!! note
+
+    You must run a beacon node and an execution client to operate a node on Mainnet post-Merge.
+    To become a validator, you must also run a validator client (either
+    [in the same process as the beacon node](../HowTo/Get-Started/Run-Teku.md#start-the-clients-in-a-single-process) or
+    [separately](../HowTo/Get-Started/Run-Teku.md#run-the-clients-separately).
+
+The PoS mechanism randomly chooses validators to propose or validate blocks on the
+[Beacon Chain](https://ethereum.org/en/upgrades/beacon-chain/) (consensus blocks) in defined time frames.
```

Comment:
Is consensus block a general Ethereum term? Otherwise I'd probably just go with block.